### ISSUE TITLE
mobile website ios: hamburger menu not working inlanding page

### ISSUE THREAD

**Issue by Ospeleo** [subsurface/subsurface#2887]
### Describe the issue:
On iOS 13.5.1, safari, iphone Xsmax, it is impossible to browse the website: the hamburger menu is not working.

We are left with the landing page with the latest version annoucement, can tap to changer langage, but that's all

**Comment by Ospeleo** [comment]
I just checked on my desktop, and if I reduce the browser window size, the hamburger menu is not working either.

**Comment by dirkhh** [comment]
Nah, that was the wrong issue that was closed. The website is still broken and likely will remain so as the underlying infrastructure is broken and I have neither time nor energy to fix it.

**Comment by joeylobianco-better** [comment]
> Nah, that was the wrong issue that was closed. The website is still broken and likely will remain so as the underlying infrastructure is broken and I have neither time nor energy to fix it.

Where is this broken infrastructure detailed out, so that others can try to fix it?

**Comment by dirkhh** [comment]
The old part of the website (mainly the landing page and a couple of older other pages and the manuals) are in a static export of an ancient broken WordPress site. All the new stuff is in GitHub pages. So what's missing is a port of the landing page to GitHub pages...
